mitosheet/src/components/taskpanes/UserDefinedImport/UserDefinedImportTaskpane.tsx

```diff
@@ -168,16 +175,32 @@ const UserDefinedImportTaskpane = (props: UserDefinedImportTaskpaneProps): JSX.E
                                         }}
                                     />
                                 )
-                            } else {
-                                // TODO: in the future, handle other types (e.g. booleans or Unions here)
+                            } else if (paramType === 'bool') {
+                                inputElement = (
+                                    <Toggle
+                                        value={paramValue.toLowerCase().includes('true')}
+                                        onChange={() => {
+                                            const newValue = !paramValue.toLowerCase().includes('true');
```

Comment:
Why aren't we just using actual True and False values here instead of checking if the param Value includes true?

Reply:
I'll lead some documentation. We want to do all the parsing in one location for all these params, and since we support all sorts of Python objects (e.g. dicts), which we can't parse on the frontend, we just do all the parsing on the backend and treat everything on the frontend as a string.